stablehlo/reference/Element.h

```diff
@@ -100,6 +100,9 @@ class Element {
   std::variant<APInt, bool, APFloat, std::pair<APFloat, APFloat>> value_;
 };
 
+/// Returns element-wise abs of Element object.
```

Comment:
I think these comments are getting redundant and doesn't provide much info. Can you delete all the comments?

Reply:
IMO we should keep the comments in the header files on the public APIs. This is very commonly followed in mlir/llvm project files [example](https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir/AsmParser/AsmParserState.h)

Reply:
Lets keep this open for @burmako .

Reply:
I think a change like that deserves a separate PR given that it's pretty sizable, and there are potential pros and cons. (Similarly to how we handled removal of braces recently).